src/System.IO.Pipes/tests/NamedPipeTests/NamedPipeTest.Simple.cs

```diff
@@ -473,12 +484,14 @@ public void OperationsOnUnconnectedClient()
         {
             using (NamedPipePair pair = CreateNamedPipePair())
             {
+                pair.Connect();
```

Comment:
@ianhays we already have test case for `InvalidOperationException` (see line 458 at this file) so could you clarify what should I do?